test/Sentry.Testing/BindableTests.cs

```diff
@@ -0,0 +1,121 @@
+#if !NETFRAMEWORK
+using Microsoft.Extensions.Configuration;
+
+namespace Sentry.Testing;
+
+public abstract class BindableTests<TOptions>(params string[] skipProperties)
```

Comment:
Not gonna lie, that class made me scratch my head in a couple of places..

Reply:
Yeah, originally the code was quite readable but there was a lot of duplication (the was a tonne of code for the unit tests themselves)... so I pulled a bunch of it back into the base class. That makes it easier to maintain (you should hardly ever have to touch the actual tests) but harder to understand... bit of a trade off. 

If this wasn't for testing, the actual test methods wouldn't exist in the derived classes (that could all be done in the base class) but then we can't surface any methods to decorate with `[Fact]` attributes... so you end up with a kind of weird base class that you have to extend in ways that wouldn't normally make sense (it's just to expose some stuff we can decorate with `[Fact]`). 

Additionally, I wanted to push some stuff into the Fixture, which can only be done in the constructor (so you can't use virtual/overloadable methods/properties for that).

Apologies, if you can find a nicer way to do it, I'm open to suggestions.